### ISSUE TITLE
Updates for Some Papers and Repositories

### ISSUE THREAD

**Issue by maygoathere** [yyyujintang/Awesome-Mamba-Papers#20]
Hello,

Thank you very much for this collection. For some of the mamba papers here from your list, could you add their corresponding repositories as well (I found them through their papers):
- STG-Mamba: Spatial-Temporal Graph Learning via Selective State Space Model: https://github.com/LincanLi98/STG-Mamba  
- Audio Mamba: Pretrained Audio State Space Model For Audio Tagging: https://github.com/diggerdu/AudioMamba
- ClinicalMamba: A Generative Clinical Language Model on Longitudinal Clinical Notes: https://github.com/whaleloops/ClinicalMamba 
- SOAR: Advancements in Small Body Object Detection for Aerial Imagery Using State Space Models and Programmable Gradients: https://github.com/yash2629/S.O.A.R 
- Audio Mamba: Pretrained Audio State Space Model For Audio Tagging: https://github.com/SarthakYadav/audio-mamba-official
- RhythmMamba: Fast Remote Physiological Measurement with Arbitrary Length Videos: https://github.com/zizheng-guo/RhythmMamba 
- Can Mamba Learn How to Learn? A Comparative Study on In-Context Learning Tasks: https://github.com/krafton-ai/mambaformer-icl 
- Audio Mamba: Bidirectional State Space Model for Audio Representation Learning: https://github.com/mhamzaerol/Audio-Mamba-AuM
- Efficient Network Traffic Classification via Pre-training Unidirectional Mamba: https://github.com/wangtz19/NetMamba 
- Dual-path Mamba: Short and Long-term Bidirectional Selective Structured State Space Models for Speech Separation: https://github.com/xi-j/Mamba-TasNet 
- Mamba-ND: Selective State Space Modeling for Multi-Dimensional Data: https://github.com/jacklishufan/Mamba-ND 

Also, in its repository, the work “Audio Mamba: Bidirectional State Space Model for Audio Representation Learning” refers to an OpenReview preprint (https://openreview.net/forum?id=RZu0ZlQIUI), which corresponds to the first release on the web. Maybe, this could also be shown in the list. 

Thank you again for this great list!

**Comment by yyyujintang** [comment]
Thank you for providing these links. I've added them.
